static/frontEndLayout.js

```diff
@@ -338,6 +338,86 @@ var buyPlot = function() {
     }
 }
 
+var createSeedInfo = function(id) {
+    for (var recipe_id in RECIPE_CONFIG) {
+        document.getElementById('seedInfoButton' + recipe_id).style.backgroundColor = 'white';
+        document.getElementById('seedInfoButton' + recipe_id).style.color = 'black';
+    }
+    document.getElementById('seedInfoButton' + id).style.backgroundColor = 'blue';
+    document.getElementById('seedInfoButton' + id).style.color = 'white';
+
+    var seedInfoMenu = document.getElementById('seedInfoContent');
+    while (seedInfoMenu.firstChild) {
+        seedInfoMenu.removeChild(seedInfoMenu.firstChild);
+    }
+    var div = document.createElement("DIV");
+        var img = document.createElement("IMG");
+            img.src = GAME_CONFIG.seeds[id].imageSmall;
+            img.style.display = 'inline';
+        var divName = document.createElement("DIV");
+            divName.innerHTML = GAME_CONFIG.seeds[id].name;
+            divName.style.display = 'inline';
+    div.appendChild(img);
+    div.appendChild(divName);
+    var divCost = document.createElement("DIV");
+        divCost.innerHTML = 'Cost:';
+
+    seedInfoMenu.appendChild(div);
+    seedInfoMenu.appendChild(divCost);
+
+    for (var cost in RECIPE_CONFIG[id]) {
+        if (RECIPE_CONFIG[id][cost] > 0) {
+            var div = document.createElement("DIV");
+                var img = document.createElement("IMG");
+                if (cost == 'cashCost') {img.src = "/static/cash.png";}
+                if (cost == 'carrotsCost') {img.src = "/static/carrot_s.png";}
+                if (cost == 'grassCost') {img.src = "/static/grass_s.png";}
+                if (cost == 'fertilizerCost') {img.src = "/static/fertilizer_s.png";}
+                img.style.height = '20px';
+                img.style.display = 'inline';
+
+                var divCost = document.createElement("DIV");
+                divCost.innerHTML = "x" + RECIPE_CONFIG[id][cost];
+                divCost.style.display = 'inline';
+            div.appendChild(img);
+            div.appendChild(divCost);
+
+            seedInfoMenu.appendChild(div);
+        }
+    }
+
+    var divYield = document.createElement("DIV");
+        divYield.innerHTML = "Yield:";
+    seedInfoMenu.appendChild(divYield);
+
+    for (var yield in GAME_CONFIG.seeds[id]) {
+        if (GAME_CONFIG.seeds[id][yield] > 0 && yield.includes("Yield")) {
```

Comment:
This is an error-prone way to find the yield of a seed.  Instead, put it in its own section of the seed definition:
```
c: {
  'name': "Carrot",
  'yield': {
    'cash': 15,
    'carrot': 5,
    'seed': 1
  }
}
```